eng/Versions.props

```diff
@@ -108,7 +108,7 @@
     <MicrosoftVisualStudioPackageLanguageService150PackageVersion>16.7.30204.53-pre</MicrosoftVisualStudioPackageLanguageService150PackageVersion>
     <MicrosoftVisualStudioLiveSharePackageVersion>0.3.1074</MicrosoftVisualStudioLiveSharePackageVersion>
     <MicrosoftVisualStudioOLEInteropPackageVersion>7.10.6071</MicrosoftVisualStudioOLEInteropPackageVersion>
-    <MicrosoftVisualStudioProjectSystemManagedVSPackageVersion>3.0.0-beta1-63607-01</MicrosoftVisualStudioProjectSystemManagedVSPackageVersion>
+    <MicrosoftVisualStudioProjectSystemManagedVSPackageVersion>16.8.1-beta1-1001-01</MicrosoftVisualStudioProjectSystemManagedVSPackageVersion>
```

Comment:
Why❔ And what are the implications of this large jump in the repo's deliverables❔

Reply:
The 3.0.0-beta1 version wasn't available after removing the feeds above. After reading your comment above, I think adding the https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json feed might help here.